Source/Container/HCList.h

```diff
@@ -108,7 +110,9 @@ HCBoolean HCListIterationHasEnded(HCListIterator* iterator);
 //----------------------------------------------------------------------------------------------------------------------------------
 // MARK: - Iteration Convenience Operations
 //----------------------------------------------------------------------------------------------------------------------------------
-void HCListForEach(HCListRef self, HCListForEachFunction forEachFunction);
-void HCListForEachWithContext(HCListRef self, HCListForEachWithContextFunction forEachFunction, void* context);
+void HCListForEach(HCListRef self, HCListForEachFunction forEachFunction, void* context);
+void HCListFilter(HCListRef self, HCListFilterFunction filterFunction, void* context);
```

Comment:
Swift calls the input function for filter `isIncluded` if you want a more descriptive name.